### ISSUE TITLE
learning rate of the embedding layer

### ISSUE THREAD

**Issue by xiaomengyc** [zcyang/imageqa-san#14]
I implemented this network in Keras, but got very bad results. After looking for the differences with your Theano codes, I found that you set the learning rate of embedding layer as a very big value 80. 
`options['w_emb_lr'] = numpy.float32(80)`
I tried to comment this line and the results of your Theano version dropped. It turns out this option is very important. Could you please explain the reason for this option?  I will be grateful if you could give a suggestion of a paper related to this issue!
Thank you very much!
